apps/molgenis-components/src/components/forms/EditModalWizard.vue

```diff
@@ -66,30 +66,36 @@ export default {
   },
   mounted() {
     this.columnsSplitByHeadings = splitColumnsByHeadings(
-      this.tableMetaData.columns,
-      this.visibleColumns
+      filterVisibleColumns(this.tableMetaData.columns, this.visibleColumns)
     );
     this.$emit("setPageCount", this.columnsSplitByHeadings.length);
   },
   emits: ["setPageCount", "update:modelValue"],
 };
 
-function splitColumnsByHeadings(columns, visibleColumns) {
+function filterVisibleColumns(columns, visibleColumns) {
+  if (!visibleColumns) {
+    return columns;
+  } else {
+    return columns.filter(
+      (column) =>
+        !visibleColumns ||
+        visibleColumns.find(
+          (visibleColumn) => column.name === visibleColumn.name
```

Comment:
maybe more readable ( and faster ) to map visibleColumn to name list and use .includes

Reply:
oke lol nevermind , i was looking at an outdated diff